sql/hivecontext-compatibility/src/test/scala/org/apache/spark/sql/hive/HiveContextCompatibilitySuite.scala

```diff
@@ -99,4 +105,41 @@ class HiveContextCompatibilitySuite extends SparkFunSuite with BeforeAndAfterEac
     assert(databases3.toSeq == Seq("default"))
   }
 
+  test("check change after refresh") {
```

Comment:
Looks like `RefreshTable` command is actually doing more work. I think we need to make `RefreshTable` and `sparkSession.catalog.refreshTable` have the same behavior. Can you make that change?